accelerator/abstract_accelerator.py

```diff
@@ -17,6 +17,18 @@ def __init__(self):
     def is_synchronized_device(self):
         ...
 
+    @abc.abstractmethod
+    def use_host_timers(self):
+        ...
+
+    @abc.abstractmethod
+    def has_data_dependency_resolving(self):
```

Comment:
How about simplifying name with:

```suggestion
    def can_resolve_data_dependency(self):
```

or 
```suggestion
    def resolves_dependency(self):
```